Rakefile

```diff
@@ -18,7 +18,7 @@ require_relative 'tasks/assets'
 require_relative 'tasks/profiles'
 require_relative 'tasks/geo'
 
-Mongoid.load!('./config/mongoid.yml')
+Mongoid.load!('./config/mongoid.yml', :test)
```

Comment:
If I remove `:test`, the `environment` is not set:

```
$ rake test
rake aborted!
Mongoid::Errors::NoEnvironment: 
Problem:
  Could not load the configuration since no environment was defined.
Summary:
  Mongoid attempted to find the appropriate environment but no Rails.env, Sinatra::Base.environment, RACK_ENV, or MONGOID_ENV could be found.
Resolution:
  Make sure some environment is set from the mentioned options. Mongoid cannot load configuration from the yaml without knowing which environment it is in, and we have considered defaulting to development an undesireable side effect of this not being defined.
```

 I can also get it to work by doing `export MONGOID_ENV=test` before running `rake test`.

How is the proper way to manage the `environment`?